storage/google/cloud/storage/blob.py

```diff
@@ -1983,7 +1984,7 @@ def _get_encryption_headers(key, source=False):
     }
 
 
-def _quote(value):
+def _quote(value, safe=b"~"):
```

Comment:
Why not include `/` as well?

Reply:
@frankyn Because it is not always wanted:  when quoting for the JSON API, we want to percent-escape slashes.  Only for the XML API (`public_url`, `generate_*_signed_url`) do we want to avoid escaping slashes.

Reply:
Thanks for clarifying